From 0e2aa06c47b7291239bec3de673b80097e2e3a3b Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi2@arcor.de>
Date: Sun, 14 May 2017 16:24:09 +0200
Subject: [PATCH 21/74] Updated to 14-May-17 14:24:00 UTC

 - Added LineageOS/android_device_xiaomi_lithium
 - Added LineageOS/android_device_xiaomi_scorpio

Change-Id: I0f9fcd1f2098a0b4e13c30fc5900912960e2365d
---
 default.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/default.xml b/default.xml
index a3b6f30..cca4a0c 100644
--- a/default.xml
+++ b/default.xml
@@ -745,9 +745,11 @@
   <project name="LineageOS/android_device_xiaomi_ido" />
   <project name="LineageOS/android_device_xiaomi_kenzo" />
   <project name="LineageOS/android_device_xiaomi_libra" />
+  <project name="LineageOS/android_device_xiaomi_lithium" />
   <project name="LineageOS/android_device_xiaomi_msm8956-common" />
   <project name="LineageOS/android_device_xiaomi_msm8996-common" />
   <project name="LineageOS/android_device_xiaomi_natrium" />
+  <project name="LineageOS/android_device_xiaomi_scorpio" />
   <project name="LineageOS/android_device_yuhua_x2" />
   <project name="LineageOS/android_device_yu_jalebi" />
   <project name="LineageOS/android_device_yu_lettuce" />
-- 
2.17.1

